openassessment/templates/openassessmentblock/response/oa_response_graded.html

```diff
@@ -26,9 +25,7 @@
     <div class="wrapper--step__content">
         <div class="step__content">
             <article class="submission__answer__display">
-                <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
-
-                {% trans "Your response to the question above:" as translated_label %}
+                {% trans "Your response" as translated_label %}
```

Comment:
This header level is defined in the template on the next line.  This can be seen on the File Upload ORA problems.  I did catch a missed header from the translated_header right below this one.